android/sdl_android/src/androidTest/java/com/smartdevicelink/test/rpc/datatypes/TireStatusTest.java

```diff
@@ -23,7 +23,7 @@
     @Override
     public void setUp() {
         msg = new TireStatus();
-        msg.setPressureTellTale(TestValues.GENERAL_WARNINGLIGHTSTATUS);
+        msg.setPressureTelltale(TestValues.GENERAL_WARNINGLIGHTSTATUS);
```

Comment:
@kostyaBoss I would suggest reverting these changes in `TireStatusTest.java`.  These are valid changes, since the `pressureTellTale` functions are being renamed per the parameter, but is out of scope of the PR.